src/Microsoft.Azure.SignalR/HubHost/NegotiateHandler.cs

```diff
@@ -63,19 +66,23 @@ public NegotiationResponse Process(HttpContext context, string hubName)
             };
         }
 
-        private string GetQueryString(string originalQueryString)
+        private string GetQueryString(string originalQueryString, string cultureName)
```

Comment:
>cultureName [](start = 73, length = 11)

how about using the same pattern as ConnectionRequestIdProvider that we provide the default provider to get culture from IRequestCultureFeature?

Reply:
It seems a little complicated to use provider way if it's internal only and is rely on request culture anyway. On the other hand, if customer could use AspNetCore request culture support there seems no need to use ours. So I think current implementation should be OK.